Tools/AP_Periph/can.cpp

```diff
@@ -583,6 +583,32 @@ static void set_rgb_led(uint8_t red, uint8_t green, uint8_t blue)
         dev->transfer(&v, 1, nullptr, 0);
     }
 #endif // HAL_PERIPH_ENABLE_NCP5623_BGR_LED
+#ifdef HAL_PERIPH_ENABLE_TOSHIBA_LED
+{
+#define TOSHIBA_LED_PWM0    0x01    // pwm0 register
+#define TOSHIBA_LED_ENABLE  0x04    // enable register
+#define TOSHIBA_LED_I2C_ADDR 0x55    // default I2C bus address
+
+        static AP_HAL::OwnPtr<AP_HAL::I2CDevice> dev;
+        if (!dev) {
+            dev = std::move(hal.i2c_mgr->get_device(0, TOSHIBA_LED_I2C_ADDR));
+        }
+        WITH_SEMAPHORE(dev->get_semaphore());
+        dev->set_retries(10);
+
+        // enable the led
+        dev->write_register(TOSHIBA_LED_ENABLE, 0x03);
```

Comment:
Does this really need to be called on every led update? Seems like excessive i2c transactions. This should go either up into where dev gets set or in a ```static bool run_once;`` thing right there.

Reply:
in AP_Notify, there is an init sequence that sets that register, then retries drops to 1 within the same init. `cann.cpp` is all static functions. I looked at moving that inside the lazy load, but was unaware of the consequences if having it set to 10. I will move.